includes/Parameters.php

```diff
@@ -832,7 +832,7 @@ public function _title( $option ) {
 	 */
 	public function _titleregexp( $option ) {
 		$data = $this->getParameter( 'title' );
-		if ( !isset( $data['regexp'] ) || !is_array( $data['regexp'] ) ) {
+		if ( !is_array( $data['regexp'] ?? false ) ) {
```

Comment:
Don't do this part. I am doing another patch right now to fix all of this type of stuff.